tests/src/main/java/org/neo4j/graphalgo/core/WeightMappingSerialization.java

```diff
@@ -5,6 +5,7 @@
 import com.carrotsearch.hppc.cursors.LongDoubleCursor;
 import org.apache.lucene.store.DataInput;
```

Comment:
Yes. It's only for File serialization and we use the vint encoding to reduce file size by quite a bit (2x-3x on the wikipedia graph). In the future, we can copy the vint encoding into the repo and not use lucene's classes directly here.